.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Fixes (issue #)
+
+<!--
+Please include a summary of what you want to achieve in this pull request. Remember to indicate the affected package(s).
+-->
+
+## Type of change
+
+<!--
+If this pull request changes multiple packages, please indicate the type of change for each package.
+
+If this is a new package, you may disregard this section.
+
+Please delete options that are not relevant.
+-->
+
+- [ ] <!--Package Name--> Patch: Bug fix (non-breaking change which fixes an issue)
+- [ ] <!--Package Name--> Minor: New feature (non-breaking change which adds functionality)
+- [ ] <!--Package Name--> Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] <!--Package Name--> Documenation: Only updates documentation files
```

Comment:
I think documentation should be a `Patch`. lerna will force us to make update no matter what.